packages/theme/src/colors.ts

```diff
@@ -75,10 +76,10 @@ const light = {
     backgroundSurfaceElevation2: palette.lightGray100,
     backgroundSurfaceElevation3: palette.lightWhiteAlpha1000,
     backgroundSurfaceElevationNegative: palette.lightGray200,
-    backgroundTertiaryDefaultOnElevation0: palette.lightGray200,
+    backgroundTertiaryDefaultOnElevation0: hexToRgba(palette.lightGray1000, 0.12),
```

Comment:
What's the reason behind this? Why is there alpha channel only on some of elevations? Some of the colors in pallete.ts are already using alpha channel in hex format (like lightWhiteAlpha300). Is it a good idea to introduce a concept like this in what should IMO be a simple mapping? I'd be very confused seeing this file for the first time:))